### PR TITLE
Update Nuxt to 3.11.2

### PR DESCRIPTION
**Pull request by dylankelly**
**Issue**:

### What I did

- 
- 

### How to test

- 
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: docs/components/content/DocsContentNavigation.vue [hunk 1]
```diff
@@ -50,5 +50,6 @@ const processedNav = navigation?.map((level1Item) => {
 
 .docs-section-link {
   margin-top: var(--rpl-sp-3);
+  color: var(--rpl-clr-black);
```
Comment (dylankelly) [inline docs/components/content/DocsContentNavigation.vue:53]: removing tailwind required some styling changes

File: docs/components/content/Prose/ProseH2.vue [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<template>
```
Comment (dylankelly) [inline docs/components/content/Prose/ProseH2.vue:1]: removing tailwind, we needed to define our own header renderers

File: packages/ripple-storybook/.storybook/main.ts [hunk 3]
```diff
@@ -48,9 +48,6 @@ const config: StorybookConfig = {
     name: '@storybook/vue3-vite',
     options: {}
   },
-  features: {
```
Comment (dylankelly) [inline packages/ripple-storybook/.storybook/main.ts:51]: This was previously required for docs site to work as they shared mdx dependencies. Now no longer needed.

File: packages/ripple-test-utils/step_definitions/common/mocks.ts [hunk 4]
```diff
@@ -61,7 +61,9 @@ Given(
         route: '/api/tide/page',
         status,
         response,
-        query: `?path=${path}&site=${Cypress.env('NUXT_PUBLIC_TIDE_SITE')}`
+        query: `?path=${encodeURIComponent(path)}&site=${Cypress.env(
+          'NUXT_PUBLIC_TIDE_SITE'
```
Comment (dylankelly) [inline packages/ripple-test-utils/step_definitions/common/mocks.ts:65]: This is required due to Nuxt useFetch now encoding url params automatically

File: packages/ripple-tide-search/components/global/TideCustomCollection.vue [hunk 5]
```diff
@@ -546,7 +547,7 @@ const reverseFields = computed(
 
     <template v-if="activeTab === 'map'">
       <TideSearchListingResultsMap
-        v-if="mapFeatures"
+        v-if="mapFeatures && firstLoad"
```
Comment (dylankelly) [inline packages/ripple-tide-search/components/global/TideCustomCollection.vue:550]: Adding a wait here to ensure the map data is actually loaded before attempting to render it

File: packages/nuxt-ripple-cli/src/commands/add/_templates/add/ui-lib/package.json.t [hunk 6]
```diff
@@ -42,14 +42,16 @@ to: package.json
     "test:generate-output": "jest --json --outputFile=.jest-test-results.json"
   },
   "dependencies": {
-    "@nuxt/kit": "3.3.2",
+    "@nuxt/kit": "^3.10.2",
```
Comment (waitingallday) [inline packages/nuxt-ripple-cli/src/commands/add/_templates/add/ui-lib/package.json.t:45]: Probably doesn't matter but should this be 3.11.2

File: package.json [hunk 7]
```diff
@@ -69,7 +68,6 @@
     "babel-plugin-transform-vue-jsx": "^3.7.0",
     "changelogen": "^0.5.3",
     "cypress": "12.8.1",
```
Comment (waitingallday) [inline package.json:70]: Cypress needs to be updated here too

Comment (waitingallday) [reply package.json:70]: I can't add a comment on the line but there's an override for cypress here in the root package as well that needs to be updated too

**Review by waitingallday — APPROVED** [review]

**Review by lambry — APPROVED** [review]

**Pull request merged**